grafana/build.yml

```diff
@@ -3,3 +3,8 @@ variants:
   - tag: latest
     aliases:
       - :4.0.0-1.1.1
+  - tag: app
+    aliases:
+      - :4.0.0-1.2.0
```

Comment:
Actually, thinking about this more, we should put an extra suffix on the alias, probably` - 4.0.0-1.2.0-app`. We can remove the suffix once we move to the branch permanently.